Examples/Io/Root/src/RootTrackSummaryWriter.cpp

```diff
@@ -409,7 +409,8 @@ ProcessCode RootTrackSummaryWriter::writeT(const AlgorithmContext& ctx,
 
       const auto& covariance = track.covariance();
       for (unsigned int i = 0; i < Acts::eBoundSize; ++i) {
-        error[i] = std::sqrt(covariance(i, i));
+        error[i] =
+            std::sqrt(covariance(i, i));  // MARK: fpeMask(FLTINV, 1, #3999)
```

Comment:
the way I handled this in the other writer is to check if it is lower than `0` and if so write `nan`

Reply:
good point to have this aligned.